pkg/controller/ipam/antrea_ipam_controller_test.go

```diff
@@ -231,17 +202,13 @@ func TestReleaseStaleAddresses(t *testing.T) {
 			Phase: crdv1a2.IPAddressPhasePreallocated,
```

Comment:
Could you change all `IPAddressPhasePreallocated` to `IPAddressPhaseReserved`

Reply:
Done